statesman-engine/src/main/java/io/appform/statesman/engine/handlebars/HandleBarsHelperRegistry.java

```diff
@@ -354,6 +347,42 @@ private void registerAdd() {
         });
     }
 
+    /*
+     * computes no. of days from the given date till the current date.
+     * {{computeDays dateNode 'dd MM yyyy'}}
+     */
+    private void registerComputeDays() {
+        handlebars.registerHelper("computeDays", (String context, Options options) -> {
+            try {
+                if (null == options.params || options.params.length == 0) {
+                    return 0L;
+                }
+                if (null == context || Strings.isNullOrEmpty(context)) {
+                    return 0L;
+                }
+                String dateFormat = options.param(0);
+                if(dateFormat == null || Strings.isNullOrEmpty(dateFormat)) {
+                    return 0L;
+                }
```

Comment:
Make this optional. Assume a standard date format. This will help keep your templates brief.

Reply:
done.